source/wp-content/themes/wporg-developer-2023/src/chapter-list/style.scss

```diff
@@ -3,6 +3,35 @@
 	--local--icon-size: calc(var(--local--line-height) * 1em);
 	line-height: var(--local--line-height);
 
+	@media (max-width: 767px) {
+		border: 1px solid var(--wp--preset--color--light-grey-1);
+		padding: 15px var(--wp--preset--spacing--20);
+
+		.wporg-chapter-list__list {
```

Comment:
Did we agree on `__` as convention? If yes, no reply is necessary.

Reply:
Not that I'm aware of, but felt it was set by prior art https://github.com/WordPress/wporg-mu-plugins/blob/trunk/mu-plugins/blocks/table-of-contents/index.php#L60